spec/orm/activerecord_spec.rb

```diff
@@ -672,7 +672,7 @@ def filename
     end
 
     after do
-      FileUtils.rm_rf(public_path("uploads"))
+      FileUtils.rm_rf(file_path("uploads"))
```

Comment:
The change from `public_path` to `file_path`  need to be restored back. As stated in #2209,

> Changes in spec/orm/activerecord_spec.rb:
>
> - Add tests which covers this flows
> - replace cleaning folder from 'file_path' to 'public_path'. Cause file_path is 'spec/fixtures'
>
> and we don't create any 'uploads' folders there. We works with 'spec/public/uploads' folder.

Reply:
I don't understand this. `file_path` would point to `spec/fixtures/uploads`, and `public_path` points to `spec/public/uploads`. 

So, if I understand correctly, `public_path` is the final destination, and thus is the path to use.

Reply:
> file_path would point to spec/fixtures/uploads, and public_path points to spec/public/uploads

That's correct, and we don't create anything in spec/fixtures/uploads. During test execution, files are stored in spec/public/uploads so it need to be cleaned up.

> public_path is the final destination, and thus is the path to use.

That's also true, but you're trying to change from `public_path("uploads")` to `file_path("uploads")`, which results in leaving junks in spec/public/uploads.

Reply:
I don't understand. Was this part wrong before #2209 ? Because this restores the behavior prior to that commit.

Reply:
> Was this part wrong before #2209 ?

Yes, in that sense #2209 consists of 2 fixes(of course this is not ideal).

Let me illustrate how the issue leads to incorrect result. There's a spec here:

https://github.com/carrierwaveuploader/carrierwave/blob/d50d80eee95054d4e2567b685c5b82a8128146f7/spec/orm/activerecord_spec.rb#L686-L692

and if you comment out the line 689, the spec naturally fails since you have skipped storing files.
But if you use `FileUtils.rm_rf(file_path("uploads"))` instead of `public_path` here:

https://github.com/carrierwaveuploader/carrierwave/blob/d50d80eee95054d4e2567b685c5b82a8128146f7/spec/orm/activerecord_spec.rb#L674-L676

the spec no longer fails because those files are left undeleted from the previous example execution.

Reply:
Ah, I understand now. Thank you for your patience :)

I have updated the PR.